plotter/plot_classes/roc_plot.py

```diff
@@ -64,5 +68,52 @@ def plot(self):
                 )
                 roc_plot.set_ratio_class(1, "qcd")
 
+                # add cut values and background rejections if desired
+                if self.config.show_cuts == True:
+                    # calculate the efficiencies for the specific cut values
+                    if self.config.cut_values is not None:
+                        cut_values = self.config.cut_values
+
+                    sig_disc = df[is_hs][pDisp]     # convenient to store signal discriminants
+                    N_signal = len(sig_disc)
+
+                    cut_effs = []   # initialize array to store efficiencies from cut values
+                    cut_rejs = []   # initialize array to store corresponding bkg effs from cuts
+
+                    for cut in cut_values:
+                        true_pos = sig_disc[sig_disc >= cut]    # determine the signal that passes the cut
+                        eff_ = len(true_pos)/N_signal
+                        rej_ = calc_rej(
+                            df[is_hs][pDisp].values, df[is_pu][pDisp].values, eff_
+                        )
+                        cut_effs.append(eff_)
+                        cut_rejs.append(rej_)
+
+                    # plot the cuts and the corresponding bkg rejs
+                    markers = ['o', 'v', 's', 'P', 'X', 'd']
+                    for i in range(len(cut_values)):
+                        roc_plot.axis_top.scatter(
+                            cut_effs[i], 
+                            cut_rejs[i], 
+                            s = 75,
+                            marker = markers[i],
+                            facecolors = list(roc_plot.label_colours.values())[-1],
+                            edgecolors = 'black',
+                            alpha=0.6,
+                            label = "cut = {0:.3f}, rej. = {1:.2e}".format(cut_values[i], cut_rejs[i]),
+                            zorder = 99
+                        )
```

Comment:
maybe add an assertion here that `len(cut_values) <= len(markers)`? otherwise this could fail with an unclear error

Reply:
Okay, will do. I just assumed more than 6 cuts would be plotted.